REST/RESTListener+Replicate.cc

```diff
@@ -289,7 +317,13 @@ namespace litecore { namespace REST {
                          canceled ? "Stopped" : "No matching task");
             return;
         }
-
+        // Auth:
+        slice authType = params["authtype"].asString();
```

Comment:
This isn't compatible with the [CouchDB REST API](https://docs.couchdb.org/en/stable/api/server/common.html#api-server-replicate). Unfortunately the API docs don't describe how to provide credentials; it's been years since I worked with it but IIRC they're either encoded in the URL (which our URL parser doesn't handle), or else the `source` and `target` can be a dict instead of a string and the credentials are keys in that dict.

At this level I'm OK with not going by the spec. But don't include the `authtype` parameter, since all you're handling is Basic auth anyway. Just look for the presence of `user`.

Reply:
Okay, take it.